### DISCUSSION TITLE
Issues with FS.

### DISCUSSION THREAD

**Discussion by S10MC2015** [pyodide/pyodide#2184] [Q&A] [answered]
This worked completely fine yesterday but now I get this.
![image](https://user-images.githubusercontent.com/30575036/154681821-3c269d9a-9376-423c-bf2e-29d5fa97034f.png)
(using v0.19)

(PS also in Dev is it intended that instead of "Skipping unknown package" it just thows an exception instead)

**Comment by ryanking13** [comment, marked as answer]
Could you provide some more information and a code that we can reproduce the error?

**Reply by S10MC2015** [reply]
![image](https://user-images.githubusercontent.com/30575036/154686226-8b704f5f-371c-4bb2-8b89-a7f307c8690c.png)

**Reply by S10MC2015** [reply]
![image](https://user-images.githubusercontent.com/30575036/154686885-9a5c2eb4-cf12-4534-bdb8-343b2d8eeffd.png)
better example

**Reply by ryanking13** [reply]
Maybe your path to a file is wrong. Try `filename = "asdf.epub"` not `filename = "/asdf.epub"`.

Note that you can check your current working directory with `os.getcwd()`, it might be `/home/pyodide`, not `/`.
